packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -103,23 +90,29 @@ Future<List<Plugin>> findPlugins(
     logger: globals.logger,
     throwOnError: throwOnError,
   );
-  final Set<String> devDependencies;
-  if (!determineDevDependencies) {
-    devDependencies = <String>{};
-  } else {
-    devDependencies = await computeExclusiveDevDependencies(
-      pub,
-      logger: globals.logger,
-      project: project,
-    );
-  }
-  for (final Package package in packageConfig.packages) {
-    final Uri packageRoot = package.packageUriRoot.resolve('..');
+  final Map<String, Dependency> transitiveDependencies = computeTransitiveDependencies(
```

Comment:
This is a change in behavior: previously unless `determineDevDependencies` we would not do any work.

It might be fine to make this unconditional, but just pointing that out if that was not your intent (the flag that flips on `determineDevDependencies` is _not `true` yet).